modular_citadel/code/modules/reagents/objects/items.dm

```diff
@@ -107,6 +107,7 @@
 		if(-INFINITY to 1)
 			color = "#c6040c"
 	desc += " The paper looks to be around a pH of [round(cont.reagents.pH, 1)]"
+	name = "used " + name
```

Comment:
```suggestion
	name = "used [name]"
```
Little peeve, generally safer to use, even though names should always be text.

Reply:
gotcha, fixed